enterprise/ha/src/test/java/jmx/HaBeanIT.java

```diff
@@ -111,7 +84,7 @@ public HighAvailability ha( HighlyAvailableGraphDatabase db )
     @Test
     public void canGetHaBean() throws Throwable
     {
-        startCluster( 1 );
+        ManagedCluster cluster = clusterRule.startCluster();
```

Comment:
`clusterRule.startCluster` gives a wrong impression that we starting cluster all over again, but not critical to fix here...